cocos/renderer/CCTexture2D.cpp

```diff
@@ -495,6 +501,11 @@ GLuint Texture2D::getName() const
     return _name;
 }
 
+GLuint Texture2D::getAlphaName() const
+{
+    return _alphaTexture == nullptr ? 0 : _alphaTexture->getName();
+}
```

Comment:
getAlphaName seems strange, i don't know it returns alpha texure's name. But i don't have better suggestion.